Makefile

```diff
@@ -118,6 +122,13 @@ test: test-unit ## Run the tests
 test-markdown test/markdown:
 	./hack/ci/marker
 
+.PHONY: test-coverage
+test-coverage:  ## Run coveralls
+	echo "mode: count" > coverage-all.out
+	$(foreach pkg,$(PKGS_COVER),\
+		go test -failfast -tags=integration -coverprofile=coverage.out -covermode=count $(pkg) || exit 1;\
+		tail -n +2 coverage.out >> coverage-all.out;)
```

Comment:
Because of the `exit 1` I think this might work. If we find that tests are not failing builds it might be because of this target. I had to not use foreach in the broker's makefile https://github.com/openshift/ansible-service-broker/commit/cc9798c432dca0177d4ed2c398eeb1aea6e1e3e6#diff-b67911656ef5d18c4ae36cb6741b7965

But I didn't have the exit 1 so that might make a difference.